action.yml

```diff
@@ -60,7 +60,7 @@ runs:
     - name: Install markbind-cli@master
       if: inputs.version == 'development'
       run: |
-        npm i -g npm@8.3.1
```

Comment:
same here

Reply:
Sorry just checked this is deploying ok without the install